AdCOM v1.0 FINAL.md

```diff
@@ -4036,6 +4036,49 @@ The following table lists the types of feeds for audio.
   </tr>
 </table>
 
+### List: ID Match Methods <a name="list_idmatchmethod"></a>
+
+The following table contains enumerations for various ways an ID could be matched to an ad request, and if they pertain to a single property or . It should be used on conjunction with the <code>mm</code> attribute in [Object: EID]
```

Comment:
@hillslatt - sentence fragment here. I think you mean "if they pertain to a single property or multiple"? Or maybe "if they pertain to a single property or app"? Also, correction to "used in conjunction with".